app/Services/NetworkDataService.php

```diff
@@ -204,4 +205,9 @@ public static function firstOrCreateNetworkAircraft(
 
         return $aircraft;
     }
+
+    private function pilotValid(array $pilot): bool
+    {
+        return preg_match('/^[0-7]{4}$/', $pilot['transponder']);
```

Comment:
Assuming you're keeping `0000` as a valid transponder code?

Reply:
(I know it is one, but I think (?) FSD uses 0000 as some invalid indicator?)

Reply:
Yeah - it technically is valid, though also not!